src/data/resolvers/task.ts

```diff
@@ -105,6 +106,7 @@ export const taskResolvers = ({
       }
       if (existingEthPotId) {
         const payoutClaimedFilter = colonyClient.filters.PayoutClaimed(
+          null,
           existingEthPotId,
```

Comment:
You can safely add a `@ts-ignore` here to deal with that typescript error.

I _think_ is didn't generate something properly in `colony-js`, but until I get to investigate this, it can stay ignore